service/worker/scanner/README.md

```diff
@@ -0,0 +1,598 @@
+# A brief* overview of Scanner and Fixer and how to control them
+
+There are a variety of problems with the Scanner and Fixer related code, so
```

Comment:
Maybe put a todo for the assumed audience knowledge and the why: `what is Scanner and Fixer?` and `what problmes are they intended to solve`. The rest of this doc is concerned the how.

Reply:
I wouldn't block on this though. writing the purpose and intent of scanner and fixer are out of the scope of this PR

Reply:
I would call out maybe in words of fire though that these features are all essentially in pre-release/alpha

Reply:
like scanner/fixer as a whole?